docs/wasmer-js/node-modules/examples/hello-world/hello-world.md

```diff
@@ -1,118 +1,199 @@
 ---
-id: wasmer-js-node-modules-hello-world
-title: Wasmer-JS Node Modules Hello World
+id: wasmer-js-modules-hello-world
+title: Wasmer-JS Modules in the Browser
 sidebar_label: Hello World
 ---
 
 [Full Example Project Source Code](https://github.com/wasmerio/docs.wasmer.io/tree/master/docs/wasmer-js/node-modules/examples/hello-world)
 
-In this example, we will run the WASI module [as-echo](https://github.com/torch2424/as-echo), using `@wasmer/wasi`. 
+# Hello World!
 
-The WASI module will echo "Hello World", and we will get the standard output from `/dev/stdout` using the `@wasmer/wasmfs` module. 
+In this very simple example, we want to develop a Browser-based application that calls a WebAssembly module that in turn calls a native "OS" function.
```

Comment:
So I try to avoid using words like "simple", as it can be discouraging for developers when something is "simple", but they are having a hard time with it 😄

Not a blocker, but something we can go back in another PR and fix 👍

Reply:
Hmmm....

If a concept is described as "simple", yet people struggle to understand it, then I would contend that the explanation is at fault because it has failed to transmit the underlying simplicity - as Albert Einstein said *"If you can't explain it to a six year old, then you don't understand it yourself"*

Further to this, I would also contend that there exists the widespread misunderstanding that the words "simple" and "easy" are synonyms.  This is simply not true and is the basis for a lot of people giving up after only a few steps of their learning journey.  "Simple" means minimum complexity, "Easy" means minimum effort - the two are ***not*** the same (See Rich Hickey's excellent explanation of this https://www.youtube.com/watch?v=oytL881p-nQ)

That said, I'm happy to swap the word "simple" for "basic" and then ensure that the explanations do not require the reader to make any leaps of understanding.

Reply:
Ah, so I hope you don't mind these discussions. Just for a quick check in, totally am a super friendly person, and again am so happy to have your contributions, and be able to discuss this 😄 

So some quick background on me: I don't have a ton of experience in the education field, but I've been really passionate about writing documentation for the past few years, so I've been reaching out for all the resources I can on how / what makes good documentation good, and bad documentation bad. So our discussions have been immensely helpful for me, thank you 😄 And like not just saying that to be nice, or like because I work for the company, but like for real personally saying that 👍 

That being said, I do think you make some very good points:

> If a concept is described as "simple", yet people struggle to understand it, then I would contend that the explanation is at fault because it has failed to transmit the underlying simplicity - as Albert Einstein said "If you can't explain it to a six year old, then you don't understand it yourself"

Using the word "simple" on a complex explanation definitely is the fault of the explanation. Let's say this is Point 1.

> Further to this, I would also contend that there exists the widespread misunderstanding that the words "simple" and "easy" are synonyms. This is simply not true and is the basis for a lot of people giving up after only a few steps of their learning journey. "Simple" means minimum complexity, "Easy" means minimum effort - the two are not the same (See Rich Hickey's excellent explanation of this https://www.youtube.com/watch?v=oytL881p-nQ)

I also agree here, "Simple" and "Easy" do not mean the same thing. Let's consider this Point 2.

> That said, I'm happy to swap the word "simple" for "basic" and then ensure that the explanations do not require the reader to make any leaps of understanding.

So I think I have a suggestion along the same line, but without using another "similar" word 😄 

So from my experience of learning how to write good docs, a friend of mine, Carolyn Stransky, gave a really good talk on "Humanizing your documentation". In the talk, she mentioned [using the words "simple" or "easy"](https://youtu.be/FvPXMUuUCS4?t=1244) is bad practice. And I've seen this echoed in multiple talks in the JS, and general development, community. This is because, I try to write documentation from the audience's point of view, and our audience can be: people of different skill levels, people who are already frustrated looking for additional help, and people who may not be able to understand english. And like you mentioned in point 2, even though it is wrong, people think simple and easy are the same thing, so it can be discouraging or frustrating if your a junior developer who doesn't understand, a senior developer who spent the past 2 hours debugging and is now trying to understand why it doesn't work, or someone trying to "google translate" the docs to their native language.

So as Carolyn stated in their talk, maybe instead of just avoiding the word "simple", maybe we can say something like: "In this example, we will to develop a Browser-based application that calls a WebAssembly module. This in turn calls a native "OS" function, to output the string "Hello World"".

What do you think? 😄

Reply:
Totally with you on this...

I've spent the last 25 years writing stuff like software training courses, technical documentation, trade-show presentations (even written two books) and in all this, I am continually striving to meet the three criteria of:

* Simplicity  (as per the above definition of "simple")
* Clarity
* Brevity

So I totally agree with your motivation and *modus operandi*, albeit, we have some differences in terminology.

Here's a suggestion, seeing as its 8:30PM here in the YooKay, could you merge the changes for https://github.com/wasmerio/docs.wasmer.io/pull/25, then on Monday, I'll have a shiny new PR waiting for you that removes words like "basic" etc.  The reason for a new PR is that I've seen a few more annoying typos and clumsy phrasing that also need to be smoothed out...

What say ye?

Reply:
@ChrisWhealy That sounds perfect! Thank you! 😄 I am actually testing out #25 now haha! Will merge it in like 10 minutes 😄